tenderly/unchainedProxy.ts

```diff
@@ -3,7 +3,7 @@ import dotenv from 'dotenv'
 import express from 'express'
 import { readFileSync } from 'fs'
 import { createProxyMiddleware, fixRequestBody } from 'http-proxy-middleware'
-import Web3 from 'web3'
+import { Web3 } from 'web3'
```

Comment:
Keeping this one here in case we need this - we need web3 to make JSON-RPC calls here, see https://github.com/wagmi-dev/viem/discussions/439

Reply:
Note, this will use the transient dep we aleady have from @ledgerhq/live-common so this has no performance impact